src/components/auto-resize-text-area.tsx

```diff
@@ -57,6 +57,7 @@ export function AutoResizeTextArea({
           setIsSubmitting(true);
           const newUuid = v4();
           //TODO adding conversation to localstorage there?
+          localStorage.setItem(newUuid, JSON.stringify({ prompt: text }));
```

Comment:
I think we'll need list of chats in local storage `{"chats" : chat[]}` where each one have uuid, prompt and other data related to response from api - it will allow us to generate sidebar items easily